<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="DESKTOP-CB2UJMH" failures="1" tests="1" name="Surefire test" time="51.124" errors="0" timestamp="2025-04-22T11:08:09 IST">
  <testcase classname="test.Employee_Bus_Attendant_Positive_Test" name="bus_attendant_creation" time="50.987">
    <failure type="java.lang.AssertionError" message="Success message mismatch! expected [Bus Attendant details created successfully] but found [Bus Attendant details created Successfully]">
      <![CDATA[java.lang.AssertionError: Success message mismatch! expected [Bus Attendant details created successfully] but found [Bus Attendant details created Successfully]
at pages.Employee_Bus_Attendant_Page.validate_create_success_message(Employee_Bus_Attendant_Page.java:138)
at test.Employee_Bus_Attendant_Positive_Test.bus_attendant_creation(Employee_Bus_Attendant_Positive_Test.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- bus_attendant_creation -->
  <testcase classname="test.Employee_Bus_Attendant_Positive_Test" name="update_bus_attendant_details" time="0.0">
    <ignored/>
  </testcase> <!-- update_bus_attendant_details -->
  <testcase classname="test.Employee_Bus_Attendant_Positive_Test" name="delete_bus_attendant_details" time="0.0">
    <ignored/>
  </testcase> <!-- delete_bus_attendant_details -->
  <testcase classname="test.Employee_Bus_Attendant_Positive_Test" name="search_bus_attendant" time="0.0">
    <ignored/>
  </testcase> <!-- search_bus_attendant -->
</testsuite> <!-- Surefire test -->
